### ISSUE TITLE
i2c address not being recognized?

### ISSUE THREAD

**Issue by maha-haji** [adafruit/Adafruit_CircuitPython_HTU21D#12]
Hi,

I have no issues using the HTU21D, but when I run `[hex(x) for x in i2c.scan()], it does not show up in my list of i2c device addresses (i.e. I do not see '0x40' anywhere). Running `sudo i2cdetect -y 1 ` in the command line, it does show up correctly.

Any suggestions on what I might be doing wrong?

Thanks!

**Comment by ladyada** [comment]
the sensor works fine when you use it, do other i2c items appear? what ocmputer and os are you using

**Comment by maha-haji** [comment]
yes, I also have a BMP280 that I can see the i2c address of, as well as a BNO055. I'm using a Raspberry Pi 4 Model B, running the latest Raspbian OS
